tox.ini

```diff
@@ -0,0 +1,48 @@
+[tox]
+env_list =
+    check-{style}
+    test{,-warnings}{,-cov}-xdist
+    build-{docs,dist}
+
+[testenv:check-style]
+description = check code style, e.g. with flake8
+skip_install = true
+deps =
+    ruff
+commands =
+    ruff check . {posargs}
+
+[testenv]
+description =
+    run tests
+    warnings: treating warnings as errors
+    cov: with coverage
+    xdist: using parallel processing
+package = editable
+extras = tests
```

Comment:
is it possible that this is not picking up the astroquery dependency because that is specified in setup.cfg, and this is looking for 'tests' section in pyproject.toml?

Reply:
oh whoops, I called the extra `test` instead of `tests`... fixing that now